model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - id: mobile-crash-event
+    prefix: hello
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash'.
+    attributes:
+      - ref: ios.state
+        requirement_level:
+          recommended: If the crash occurred on an iOS device.
+      - ref: android.state
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
```

Comment:
Maybe we should just call it `state` and allow the OS name to be set such as `ios` or `android`, at least is flexible enough in case a new OS appears rather than introducing attributes for every OS.